composer.json

```diff
@@ -10,5 +10,17 @@
 	},
 	"autoload": {
 		"psr-0": { "Highlight\\": "" }
-	}
+	},
+	"authors": [
+		{
+			"name": "Geert Bergman",
+			"homepage": "http://www.scrivo.org/",
+			"role": "Project Author"
+		},
+		{
+			"name": "Vladimir Jimenez",
```

Comment:
I added myself on here for the fork that I submitted to Packagist, I don't think I should belong in the official package.